helm/polaris/values.yaml

```diff
@@ -0,0 +1,366 @@
+# -- The number of replicas to deploy (horizontal scaling).
+# Beware that replicas are stateless; don't set this number > 1 when using in-memory meta store manager.
+replicaCount: 1
+
+image:
+  # -- The image repository to pull from.
+  repository: localhost:5001/polaris
+  # -- The image pull policy.
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag.
+  tag: "latest"
+  # -- The path to the directory where the polaris-server.yml file should be mounted.
+  configDir: /app/config
```

Comment:
Is this really user-settable? Could we have a reasonable default for it for all cases? I'd guess it only matters for the actual java command in the pod.

Reply:
So the current docker file will mount the config to `/app/polaris-server.yml` as part of image build (so the image itself does contains a default `polaris-server.yml` which is not ideal and use the following command to start the container:
```
CMD ["java", "-jar", "/app/polaris-service-1.0.0-all.jar", "server", "polaris-server.yml"]
```

Then from the deployment template, I am using `configDir` to overwrite the location of `polaris-server.yaml` so as an end-user, I won't need to rebuild the image each time for a single config change with following snippet ([deployment.yaml](https://github.com/polaris-catalog/polaris/blob/4598e61de892783eaab14f6d94294727e5f1f2d3/helm/polaris/templates/deployment.yaml)):
```
command: ["java"]
args: ["-jar", "/app/polaris-service-1.0.0-all.jar", "server", "{{ trimSuffix "/" .Values.image.configDir }}/polaris-server.yml"]
```

And yes, if we can agreed on a default path (which I think we should), I will then remove this one and set it in the `deployment.yaml`. @dimas-b what do you think?

Reply:
I'd just use `/app/config` everywhere in helm and remove this value (for simplicity).

Reply:
Yes, lets use this as config path. Updated.